src/utils/FeatureChecker.ts

```diff
@@ -1,6 +1,12 @@
-import { getEnv } from 'app/utils/EnvironmentUtils';
+import { getEnv, getEnvOrDefault } from 'app/utils/EnvironmentUtils';
 import { Feature } from 'app/utils/Feature';
 
 export const isFeatureEnabled = (feature: Feature): boolean => {
     return getEnv(`${feature}_FEATURE`) === '1';
 };
+
+export const getEnabledAppealReasons = (): string[] => {
```

Comment:
I do not think it is best place for that new function. Feature checker was meant to handle 0/1 feature flags while your approach is different and focused purely on appeal reasons.

I'd consider changing approach to have simple 0/1 feature flag for lines reason that will get removed in the future when feature is shipped.

Reply:
Okay, I'll rework solution and refactor accordingly. This will only work however if the assumption that only one reason journey will ever be in development at any one time holds true.

Reply:
Resolved: Switched to binary feature flag as per @n-r-b 's changes.